File: MODUL-4/python/PRAK405-2310817210007-RaymondHariyono.py
n, m = map(int, input().split())
hasil = 0
baris = 0
for i in range(1, n+1):
    for j in range(i, 0, -1):
        if j > 1:
            hasilTunggal = j * m
            baris += hasilTunggal
            print(f"({j}*{m}) + ", end="")
        elif j == 1:
            hasilTerakhir = j * m
            baris += hasilTerakhir
            print(f"({j}*{m}) = {baris}")
            hasil += baris
            baris = 0
print(hasil)

File: MODUL-4/C/PRAK405-2310817210007-RaymondHariyono.c
#include <stdio.h>

int main() {
    int n, m;
    int hasil=0, hasilBaris=0;
    scanf("%d %d", &n, &m);
    for (int i = 1; i < n+1; i++) {
        for (int j = i; j > 0; j--) {
            if (j > 1) {
                printf("(%d*%d) + ", j, m);
            }
            else if (j == 1) {
                int hasilTerakhir = (j * m);
                hasilBaris = hasilBaris + hasilTerakhir;
                printf("(%d*%d) = %d\n", j, m, hasilBaris);
                hasil = hasil + hasilBaris;
                hasilBaris = 0;
            }
        }
    }
    printf("%d", hasil);
    return 0;
}